fontforge/sfd.c

```diff
@@ -4259,6 +4361,21 @@ Undoes *SFDGetUndo( FILE *sfd, SplineChar *sc,
 		lasti = img;
 	    }
 
+	    if( !strmatch(tok,"ImageX:"))
```

Comment:
A new ImageX label is fine (with me), but there shouldn't be new tags for every format that might be supported in the future. Move the reading of the mime type up to this level and for now print an error if it's not PNG. Then you can leave the file descriptor queued up to start reading the parameters (`width` etc.). That way other formats can also use `ImageX` as long as they specify the mime type immediately afterward.

Reply:
BTW, other cases where new versions if existing keywords were introduced, they got a `2` prefix, so `Image2` would be more consistent with this.